cli/src/main/java/com/box/l10n/mojito/cli/command/GitBlameCommand.java

```diff
@@ -225,7 +225,7 @@ private void getBlameResults(int lineNumber, BlameResult blameResultForFile, Git
 
             try {
                 gitRepository = builder
-                        .findGitDir(new File(sourceDirectoryParam))
+                        .findGitDir(new File(commandDirectories.sourceDirectoryPath.toString()))
```

Comment:
please change to use the getter: `commandDirectories.getSourceDirectoryPath().toString()`

Reply:
Oops 🤦‍♀️